docs/water-connection-api-1.0.0.yaml

```
@@ -0,0 +1,364 @@
+openapi: 3.0.3
+info:
+  version: 1.0.0
+  title: Water Services Management Specification
+  description: | 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)